### ISSUE TITLE
Make "persons" result validations easier to validate

### ISSUE THREAD

**Issue by timreyn** [thewca/worldcubeassociation.org#7166]
![image](https://user-images.githubusercontent.com/3885929/187052279-6920fae0-4eb1-4117-98b0-fa5241be4805.png)

It's a little tedious to confirm whether these are new people -- in order to check their dates of birth, I need to:
- click on registrations (in a new tab, so I don't lose my place)
- find the competitor
- click their registration
- click to edit their account
- for the other copy of the person, open a new tab and search for the WCA ID
- look at the list of competitions that person has attended
- click "edit their account"
- repeat for all competitors who may already be in the database

This page could have more information to make it easier. Suggestions:
- The WCA ID should link to the person already in the database, rather than just being plain text.
- Both the person already in the database and the person being added should list their dates of birth here, so it'll be easy to see if they're the same person.

**Comment by Jambrose777** [comment]
Hey! I agree the process and wording can be improved.

I have some concerns from a Data protection perspective. Primarily, we should add PII to emails more than we have to (ideally we would remove all PII from emails and only have that on the WCA website, but that's a long term thing). So I would be against providing the DOB in the email and even in the Warning.

What I can recommend is changing the duplicate persons warning to only trigger upon the following circumstances instead:
- The new competitor and WCAID share the exact same name and their DOBs match.
- The new competitor and WCAID share the exact same name, the WCA ID does not have a DOB attached to it, and the new competitors DOB year is after the year in the WCA ID.

I think that this would reduce the amount of these warnings given and still provide Delegates correct warnings whenever they miss a duplicate competitor. For other cases (like a similar DOB) the WRT can always investigate those / bring those concerns to the Delegate while posting results. But those cases are rare as is, so it might be best to handle them outside of these warnings.

I would like @thewca/wrt to take a look at the proposal and this and leave comments if needed.

**Comment by timreyn** [comment]
Good point, @Jambrose777 ! I agree with not putting more PII in emails; I was only thinking about the version shown on the webpage, but I guess it's not that easy to have different text in the email and on the webpage.

But I like your proposal better! I would still also add that the WCA ID should be linkified when the warning does come up.
